client-java/controller/src/main/java/org/evomaster/client/java/controller/expect/ExpectationHandler.java

```diff
@@ -0,0 +1,62 @@
+package org.evomaster.client.java.controller.expect;
+
+
+import java.lang.reflect.Executable;
+import java.lang.reflect.Method;
+
+/**
+ *  DSL (Domain Specific Language) for expectation management.
+ *  The goal is to allow checks on the truth value, results, and contents
+ *  of specific calls.
+ *  If expectations are active, a failed condition will
+ *  raise an exception.
+ *  If expectations are not active, no exception will be raised regardless
+ *  of the truth value of the condition.
+ *
+ *  WiP: A more finely grained mechanism for activation is planned.
+ *  The goal is to allow activating and deactivating specific types of
+ *  expectations.
+ *
+ */
+
+
+public class ExpectationHandler implements AggregateExpectation, IndividualExpectation {
+
+    /**
+     *
+     * @return a DSL object to handle expectation operations.
+     */
+    public static ExpectationHandler expectationHandler() {
+        return new ExpectationHandler();
+    }
+
+    private ExpectationHandler(){}
+
+    @Override
+    public IndividualExpectation expect(){
+        return this;
+    }
+
+    @Override
+    public IndividualExpectation that(Boolean active, Boolean condition){
```

Comment:
any reason for using `Boolean` instead of `boolean`? note that something like `!active` would then throw a NPE if `null`. This comment does apply to all cases in which you chose to use  `Boolean`

Reply:
Just reflex. Fixed.